### PR TITLE
Fixes to broken links and more

### PR DESCRIPTION
**Pull request by henryiii**
These fixes are based on the results of `bundle exec htmlproofer ./_site --assume-extension`. 

The remaining problems:

* A few pages require permission to view so show up here
* Lots of links to "Unknown", which will just give a 404
* ~~A missing logo file (jpg version)~~ (fixed)
* Our Vidyo room was deleted?!
* Jeff LeFevre's SkyHook PDF isn't accessible and I don't know where to find it.

```
- ./_site/about/team.html
  *  External link http://vassil.vassilev.info failed: 999 No error
  *  External link https://www.ci.uchicago.edu/profile/164 failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: SSL peer certificate or SSH remote key was not OK
  *  External link https://www.ci.uchicago.edu/profile/195 failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: SSL peer certificate or SSH remote key was not OK
- ./_site/as.html
  *  internally linking to Unknown, which does not exist (line 600)
     <a href="Unknown">Columnar Analysis Tools HATS</a>
  *  internally linking to Unknown, which does not exist (line 622)
     <a href="Unknown">Scientific Python and Uproot HATS</a>
  *  internally linking to Unknown, which does not exist (line 644)
     <a href="Unknown">Pattern matching for decay trees</a>
  *  internally linking to Unknown, which does not exist (line 666)
     <a href="Unknown">Summary of the 'Analysis Description Languages for the LHC' workshop</a>
  *  internally linking to Unknown, which does not exist (line 540)
     <a href="Unknown">Numpy, Pandas, PyROOT, and Uproot</a>
  *  internally linking to Unknown, which does not exist (line 554)
     <a href="Unknown">INFN Padova seminar</a>
  *  internally linking to Unknown, which does not exist (line 554)
     <a href="Unknown">Constraining effective field theories with machine learning</a>
- ./_site/docs/logo.html
  *  internally linking to /assets/logos/Iris-hep-6-WHITE-complete.jpg, which does not exist (line 124)
     <a href="/assets/logos/Iris-hep-6-WHITE-complete.jpg">jpg</a>
- ./_site/doma.html
  *  internally linking to Unknown, which does not exist (line 615)
     <a href="Unknown">US CMS Tier-2 Facilities May 2019 Meeting</a>
- ./_site/fellows.html
  *  External link https://vidyoportal.cern.ch/join/8ssmWH8zau failed: 404 No error
- ./_site/ia.html
  *  internally linking to Unknown, which does not exist (line 600)
     <a href="Unknown">Machine learning in high-energy particle physics experiments, from simulation, through reconstruction to physics analysis</a>
- ./_site/jobs.html
  *  External link https://careers.peopleclick.com/careerscp/client_jeffersonlab/external/jobDetails.do?functionName=getJobDetail&jobPostId=1264&localeCode=en-us failed: 404 No error
- ./_site/presentations/byinstitution.html
  *  External link https://users.soe.ucsc.edu/~jlefevre/skyhookdb/Skyhook-JeffNoah-Vault19.pdf failed: 403 No error
- ./_site/projects/madminer.html
  *  internally linking to Unknown, which does not exist (line 203)
     <a href="Unknown">INFN Padova seminar</a>
  *  internally linking to Unknown, which does not exist (line 203)
     <a href="Unknown">Constraining effective field theories with machine learning</a>
- ./_site/projects/tpc.html
  *  internally linking to Unknown, which does not exist (line 259)
     <a href="Unknown">US CMS Tier-2 Facilities May 2019 Meeting</a>
htmlproofer 3.11.0 | Error:  HTML-Proofer found 19 failures!
```

### CODE REVIEW

File: _data/people/djw8605.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ photo: /assets/images/team/Derek-Weitzel.jpg
 presentations:
   - title: "OSG Data Federation"
     date: 2019-06-11
-    url: https://indico.cern.ch/event/727208/contributions/3444610/attachments/1860176/3056847/OSG-StashCache.pdf
+    url: https://indico.cern.ch/event/727208/contributions/3444610/attachments/1860176/3057002/OSG-StashCache.pdf
```
Comment (henryiii) [inline _data/people/djw8605.yml:11]: These links really should remove the last two components - I believe adding a file (like a recording) invalidates the final number (3057002) here - and you can find a list of files if you simply truncate. I did that on Gordon's link below.

**Comment by henryiii** [comment]
> Jeff LeFevre's SkyHook PDF isn't accessible and I don't know where to find it.

@jlefevre?

**Comment by davidlange6** [comment]
hi @henryiii - this looks good to me - do you want to resolve the skyhook question in a different pr?

**Comment by henryiii** [comment]
Yes, this can be merged.

**Comment by jlefevre** [comment]
Oops sorry, somehow I did not see this comment, I fixed the permissions on that skyhook pdf, so the link works fine now.   No need to create new issue or re-open this merged issue.  Thanks.

**Pull request merged**